match2/wikis/rocketleague/brkts_wiki_specific.lua

```diff
@@ -351,17 +351,25 @@ function mapFunctions.getScoresAndWinner(map)
 			break
 		end
 	end
-	-- luacheck: push ignore
-	for scoreIndex, _ in Table.iter.spairs(indexedScores, p._placementSortFunction) do
-		map.winner = scoreIndex
-		break
+	local finished = map.finished
+	if not Logic.isEmpty(finished) then
+		finished = Logic.readBool(map.finished)
+	else
+		finished = not Logic.readBool(map.unfinished)
+	end
+	if finished then
+		-- luacheck: push ignore
+		for scoreIndex, _ in Table.iter.spairs(indexedScores, p._placementSortFunction) do
+			map.winner = scoreIndex
+			break
```

Comment:
Probably better as a function with a `return` instead of using `break`

Reply:
like 
```
function mapFunctions.getWinner(indexedScores)
	-- luacheck: push ignore
	for scoreIndex, _ in Table.iter.spairs(indexedScores, p._placementSortFunction) do
		return scoreIndex
	end
	-- luacheck: pop
end
```

Reply:
That's right

Reply:
Considering there's no null handling there, and you always take the first item, I (and luacheck too) would prefer removing the for loop

Reply:
okay i added a way we could do that with the last commit
pls check if that is what you thought of^^

Reply:
Seems fine, although I'm wondering why you made a different sorting function?

Reply:
2 reasons:
1) it errors out otherwise (the other sort function is build for the `Table.iter.spairs` usage)
2) the other sort function (used for the match placements) also takes sets into account, which is not necessary for maps, so we can kick that for those comparisons

Reply:
Cool!